modules/nf-core/seq2hla/meta.yml

```diff
@@ -0,0 +1,189 @@
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassII.HLAgenotype2digits":
+          type: file
+          description: HLA Class II 2-digit genotype results
+          pattern: "*ClassII.HLAgenotype2digits"
+          ontologies: []
+  class1_genotype_4d:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassI-class.HLAgenotype4digits":
+          type: file
+          description: HLA Class I 4-digit genotype results
+          pattern: "*ClassI-class.HLAgenotype4digits"
+          ontologies: []
+  class2_genotype_4d:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassII.HLAgenotype4digits":
+          type: file
+          description: HLA Class II 4-digit genotype results
+          pattern: "*ClassII.HLAgenotype4digits"
+          ontologies: []
+  class1_bowtielog:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassI-class.bowtielog":
+          type: file
+          description: HLA Class I Bowtie alignment log
+          pattern: "*ClassI-class.bowtielog"
+          ontologies: []
+  class2_bowtielog:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassII.bowtielog":
+          type: file
+          description: HLA Class II Bowtie alignment log
+          pattern: "*ClassII.bowtielog"
+          ontologies: []
+  class1_expression:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassI-class.expression":
+          type: file
+          description: HLA Class I expression results
+          pattern: "*ClassI-class.expression"
+          ontologies: []
+  class2_expression:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassII.expression":
+          type: file
+          description: HLA Class II expression results
+          pattern: "*ClassII.expression"
+          ontologies: []
+  class1_nonclass_genotype_2d:
+    - - meta:
+          type: map
+          description: |
+            Groovy Map containing sample information
+            e.g. `[ id:'sample1', single_end:false ]`
+      - "*ClassI-nonclass.HLAgenotype2digits":
+          type: file
+          description: HLA Class I non-classical 2-digit genotype results
+          pattern: "*ClassI-nonclass.HLAgenotype2digits"
+          ontologies: []
+  ambiguity:
```

Comment:
Why is this in a different order that the module?

Reply:
because I couldn't be bothered to rearrange everything again